homeassistant/components/automation/__init__.py

```diff
@@ -295,6 +307,14 @@ async def reload_service_handler(service_call: ServiceCall) -> None:
         schema=vol.Schema({}),
     )
 
+    async_register_admin_service(
```

Comment:
Should we make it a new service or just allow passing an ID to the other service ?